tests/stress/benchmarkstress_test.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"strconv"
+	"testing"
+)
+
+const (
+	deployCommand = "while true ; do echo LONG RUNNING SERVICE ; sleep 5 ; done"
+)
+
+func BenchmarkStress25Environments0Deploys0Services0LoadBalancers(b *testing.B) {
```

Comment:
Adding the zero-value entities to the test name seems redundant (also the word "Stress"). 

The following names seems to read better IMO:
```
Benchmark25Environments(b *testing.B) { ... }
Benchmark20Environments10Services(b *testing.B) { ... }
```

Reply:
✅